packages/syntax-core/src/SelectList/SelectList.tsx

```diff
@@ -71,13 +73,20 @@ const SelectList = ({
   size?: "sm" | "md" | "lg";
 }): ReactElement => {
   const id = useId();
+  const { isFocusVisible } = useFocusVisible();
+  const [isFocused, setIsFocused] = useState(false);
+
   return (
     <div
       className={classNames(styles.selectContainer, {
         [styles.opacityOverlay]: disabled,
       })}
     >
-      <label htmlFor={id} className={styles.outerTextContainer}>
+      <label
+        htmlFor={id}
+        className={styles.outerTextContainer}
+        onClick={(e) => e.preventDefault()}
```

Comment:
@JFChoe Don't do this, it's an accessibility issue if we don't select the linked form element (in this case the select)